server/src/api/listings.ts

```diff
@@ -29,7 +28,22 @@ listingRouter.get(
   '/',
   async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const listings = await listingService.getAllListings();
+      const listing = await listingService.getAllListings();
```

Comment:
The original variable name (listings) sounds better.

Reply:
@chshda Omg I am such a sotong, I edited the wrong variable 😭 🤣